cardano-node/app/cardano-cli.hs

```diff
@@ -461,6 +469,19 @@ parseFlag def active optname desc =
   flag def active $ long optname <> help desc
 
 
+parseTargetNodeAddress :: String -> String -> Parser NodeAddress
+parseTargetNodeAddress optname desc =
+  option
+    ( uncurry NodeAddress
+      . Control.Arrow.first cliParseHostAddress
+      . Control.Arrow.second cliParsePort
```

Comment:
looks like it is going to be "(hostname, portnumber)"
would it be possible to have it closer to: "hostname:portnumber" ?

Reply:
Yes, if such a form isn't suitable - I could change it.